vsintegration/tests/unittests/DocumentDiagnosticAnalyzerTests.fs

```diff
@@ -372,3 +376,13 @@ type T =
 
 let g (t : T) = t.Count()
             """)
+            
+    [<Test>]
+    member public this.DocumentDiagnosticsDontReportProjectErrors_Bug1596() =
```

Comment:
@dsyme If you build the project, they will show up in output pane (from MSBuild) and error pane (from Roslyn service in this change). But they won't be highlighted on source code as the bug reported.